src/main/java/org/apache/commons/daemon/support/DaemonLoader.java

```diff
@@ -112,8 +120,13 @@ public static boolean signal()
         return false;
     }
 
-    public static boolean load(String className, String args[])
-    {
+    /**
+     * Method to load class.
+     * @param className class Name.
```

Comment:
This is not helpful unless you _describe_ what the arguments are for. This kind of edits give Javadoc a bad name. There is no value here :-(